atomicfu/src/concurrentTest/kotlin/kotlinx/atomicfu/locks/CyclicBarrierTest.kt

```diff
@@ -1,3 +1,5 @@
+@file:OptIn(ExperimentalThreadBlockingApi::class)
```

Comment:
I think the opt-in should be put into `buildSrc/src/main/kotlin/kotlin-base-conventions.gradle.kts` (like `ExperimentalForeignApi` already is). Then, it should be possible to avoid adding `OptIn` everywhere throughout the library.